src/DependencyInjection/Configuration.php

```diff
@@ -61,6 +61,13 @@ public function getConfigTreeBuilder(): TreeBuilder
                         ->scalarNode('quality')->defaultValue(90)->end()
                         ->scalarNode('format')->defaultValue('jpg')->end()
                         ->scalarNode('constraint')->defaultValue(true)->end()
+                        ->scalarNode('resizer')->defaultNull()->end()
```

Comment:
Adding this fixes this error:

```
ErrorException:
Warning: Undefined array key "resizer"

  at vendor/sonata-project/media-bundle/src/Thumbnail/FormatThumbnail.php:107
  at Sonata\MediaBundle\Thumbnail\FormatThumbnail->generate()
     (vendor/sonata-project/media-bundle/src/Provider/BaseProvider.php:156)
  at Sonata\MediaBundle\Provider\BaseProvider->generateThumbnails()
     (vendor/sonata-project/media-bundle/src/Provider/FileProvider.php:149)
  at Sonata\MediaBundle\Provider\FileProvider->postPersist()
     (vendor/sonata-project/media-bundle/src/Listener/BaseMediaEventSubscriber.php:64)
  at Sonata\MediaBundle\Listener\BaseMediaEventSubscriber->postPersist()
     (vendor/symfony/doctrine-bridge/ContainerAwareEventManager.php:68)
  at Symfony\Bridge\Doctrine\ContainerAwareEventManager->dispatchEvent()
     (vendor/doctrine/orm/lib/Doctrine/ORM/Event/ListenersInvoker.php:115)
  at Doctrine\ORM\Event\ListenersInvoker->invoke()
     (vendor/doctrine/orm/lib/Doctrine/ORM/UnitOfWork.php:1153)
  at Doctrine\ORM\UnitOfWork->executeInserts()
     (vendor/doctrine/orm/lib/Doctrine/ORM/UnitOfWork.php:423)
  at Doctrine\ORM\UnitOfWork->commit()
     (vendor/doctrine/orm/lib/Doctrine/ORM/EntityManager.php:378)
  at Doctrine\ORM\EntityManager->flush()
     (vendor/sonata-project/doctrine-orm-admin-bundle/src/Model/ModelManager.php:71)
  at Sonata\DoctrineORMAdminBundle\Model\ModelManager->create()
     (vendor/sonata-project/admin-bundle/src/Admin/AbstractAdmin.php:445)
  at Sonata\AdminBundle\Admin\AbstractAdmin->create()
     (vendor/sonata-project/admin-bundle/src/Controller/CRUDController.php:535)
  at Sonata\AdminBundle\Controller\CRUDController->createAction()
     (vendor/sonata-project/media-bundle/src/Controller/MediaAdminController.php:54)
  at Sonata\MediaBundle\Controller\MediaAdminController->createAction()
     (vendor/symfony/http-kernel/HttpKernel.php:157)
  at Symfony\Component\HttpKernel\HttpKernel->handleRaw()
     (vendor/symfony/http-kernel/HttpKernel.php:79)
  at Symfony\Component\HttpKernel\HttpKernel->handle()
     (vendor/symfony/http-kernel/Kernel.php:199)
  at Symfony\Component\HttpKernel\Kernel->handle()
     (vendor/symfony/runtime/Runner/Symfony/HttpKernelRunner.php:37)
  at Symfony\Component\Runtime\Runner\Symfony\HttpKernelRunner->run()
     (vendor/autoload_runtime.php:35)
  at require_once('/usr/app/vendor/autoload_runtime.php')
     (public/index.php:7)                
```

Reply:
There are basically two kinds of formats. The ones that you add it via configuration, you can add as much as you can, and the admin format. You can change some settings about the admin format, and it will be used inside the sonata admin to display thumbnails and so.

The formats that you add were working but the admin format was missing this 2 options: resizer and resizer_options.